File: bro/src/main/java/me/xx2bab/bro/core/util/BroRuntimeLog.java
package me.xx2bab.bro.core.util;

import android.util.Log;

public class BroRuntimeLog {

    private static final String TAG = "[Bro] ";
    public static boolean logEnabled = true;

    public static void i(String msg) {
        if (logEnabled) {
            return;
        }
        Log.i(TAG, msg);
    }

    public static void e(String msg) {
        if (logEnabled) {
            return;
        }
        Log.e(TAG, msg);
    }

}

File: bro/src/main/java/me/xx2bab/bro/core/util/BroRuntimeLog.kt
package me.xx2bab.bro.core.util

import android.util.Log

object BroRuntimeLog {

    private const val TAG = "[Bro] "

    @JvmField
    var logEnabled = true

    @JvmStatic
    fun i(msg: String?) {
        if (!logEnabled) {
            return
        }
        Log.i(TAG, msg)
    }

    @JvmStatic
    fun e(msg: String?) {
        if (!logEnabled) {
            return
        }
        Log.e(TAG, msg)
    }
}